source/draw/gpu/opengl/OpenGLMonochrome.ooc

```diff
@@ -24,12 +24,15 @@ OpenGLMonochrome: class extends OpenGLPacked {
 		this init(rasterImage size, rasterImage stride, rasterImage buffer pointer, context)
 	}
 	toRasterDefault: override func -> RasterImage {
-		result := RasterMonochrome new(this size)
+		stride := this size x align(4)
```

Comment:
Maybe not. Also, it's pretty unclear why you align it with 4, do we need to comment this?